requirements.txt

```diff
@@ -11,7 +11,7 @@ bibtexparser>=0.6.2
 django-colorfield>=0.1.12
 requests>=2.13.0
 geoalchemy2<=0.4.1,>=0.3.0  # because egoio
-egoio
+git+https://github.com/openego/ego.io.git@dev
```

Comment:
The reason is because the install of their `master` with pip did not work and their `dev` branch is currently more likely to have the latest fixes.

Reply:
Was it an issue with the general installation of egoio? If so, please [open an issue](https://github.com/openego/ego.io/issues). Or was did the master version compatibility issues with our code? In this case I would like to fix them to a point where we can use the pypi-version of egoio instead of the github version. According to @gnn the installation from urls is deprecated in more recent versions of pip.

Reply:
I opened an [issue](https://github.com/openego/ego.io/issues/78) about that. I propose, that we merge the branch if this is the only issue. Then I'll open an issue (on oeplatform) to remind us to change it back to egoio (right now `pip install egoio` will not work), once the package is updated on pypi.

Reply:
> According to @gnn the installation from urls is deprecated in more recent versions of pip.

Is there a source for this information?